articles/spatial-anchors/how-tos/create-locate-anchors-unity.md

```diff
@@ -52,6 +52,7 @@ Learn more about the [CloudSpatialAnchorSession](/dotnet/api/microsoft.azure.spa
 Learn more about the [SessionConfiguration](/dotnet/api/microsoft.azure.spatialanchors.sessionconfiguration) class.
 
 ```csharp
+    this.cloudSession.Configuration.AccountId = @"MyAccountId";
```

Comment:
This is a little complicated to me as well. In the samples, we ask the user to define the domain, id, and key. In both this page, as well as [this page](https://docs.microsoft.com/en-us/azure/spatial-anchors/concepts/authentication?tabs=csharp), we just talk about the key. If we decide to make a change here, we should keep both links synchronized to avoid additional confusion (if we define all three here, then we should define all three there).